tests_and_analysis/test/data/force_constants/LZO/lzo_crystal.json

```diff
@@ -0,0 +1,182 @@
+{
```

Comment:
Is this file used for anything? Sorry I haven't been able to spot it! The same for `graphite_crystal.json`

Reply:
I missed removing this when I removed the other npy files, now removed